builtin/credential/aws/path_config_client.go

```diff
@@ -61,6 +62,12 @@ func (b *backend) pathConfigClient() *framework.Path {
 				Description: "The region ID for the sts_endpoint, if set.",
 			},
 
+			"sts_region_from_client": {
```

Comment:
Suggestions appreciated for a better name

Reply:
I like `sts_region_from_client`.

Reply:
Spit-balling here, but would a verb prefix like `use_sts_region_from_client` clarify this is a boolean option and we do not mean "give us the region where the client will be calling from"?

For example the K8s secrets engine uses `disable_local_ca_jwt` not `local_ca_jwt` & PKI uses `allow_role_ext_key_usage` not `role_ext_key_usage` so there is precedent to using something like this on the API.

Reply:
I have no objections to renaming it to `use_sts_region_from_client`, but I also want to call out that there exists other boolean fields that do not start with a verb. For example, the [LDAP Auth method](https://developer.hashicorp.com/vault/api-docs/auth/ldap#case_sensitive_names) has a few.

Reply:
@hashicorp/docs & @hashicorp/vault-education-approvers, opinions appreciated <3

Reply:
After some team feedback we've decided to name it to `use_sts_region_from_client`